libraries/standard/mqtt/include/mqtt.h

```diff
@@ -145,6 +145,13 @@ MQTTStatus_t MQTT_Subscribe( MQTTContext_t * const pContext,
 MQTTStatus_t MQTT_Publish( MQTTContext_t * const pContext,
                            const MQTTPublishInfo_t * const pPublishInfo );
 
+/**
+ * @brief Sends an MQTT ping request.
+ *
+ * @param[in] pContext Initialized and connected MQTT context.
+ *
+ * @return See #MQTTStatus_t.
```

Comment:
Looks like possible return codes are `MQTTSuccess`, `MQTTBadParameter`, `MQTTNoMemory`, and `MQTTSendFailed`

Reply:
Thanks